tests/test_halo_model.py

```diff
@@ -66,7 +68,7 @@ def test_tr_model_instances(thm):
 )
 def test_monotonic_dec(thm: TracerHaloModel, quantity):
     # Ensure it's going down (or potentially 1e-5 level numerical noise going up)
-    assert np.all(np.diff(getattr(thm, quantity)) <= 1e-5)
+    assert np.all((np.diff(getattr(thm, quantity))) <= 1e-5)
```

Comment:
i don't think this is necessary?

Reply:
yep sorry I was desperately trying to find what's wrong with the test.